uint8 thread
ATTLocalTiming local_timing
ATTGlobalTiming global_timing